GroversAlgorithm/GroversAlgorithm.ipynb

```diff
@@ -306,6 +306,15 @@
     "    You need to modify it to add the $-1$ phase to only the $|0...00\\rangle$ state instead.\n",
     "    <br/><br/>\n",
     "    Alternatively, you can use the same trick as in the oracle converter task.<br>\n",
+    "</details>\n",
+    "<br>\n",
+    "\n",
+    "<details>\n",
+    "  <summary><b>IMPORTANT : The test will pass but its not the correct implementation. For More details click here</b></summary>\n",
```

Comment:
I would move this comment to be part of the 1st note, so that the learner gets the information "let's implement a different transformation" and "beware, it is different and here are the consequences" in the same batch of hints.

Reply:
Also, the note "What we've just told you to do is not correct" looks a bit weird :-)

Reply:
I have done the changes. Also I added the mathematical part in Hint3. Let me know if any further changes need to be done.